etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/ruleengine/transformation/custom/RemoveObservationRequests.java

```diff
@@ -22,10 +22,13 @@
 public class RemoveObservationRequests implements CustomFhirTransformation {
 
     @Override
-    public void transform(FhirResource<?> resource, Map<String, String> args) {
+    public void transform(FhirResource<?> resource, Map<String, Object> args) {
         Bundle bundle = (Bundle) resource.getUnderlyingResource();
+        String universalServiceIdentifier =
+                args.get("universalServiceIdentifier") instanceof String
+                        ? (String) args.get("universalServiceIdentifier")
+                        : "";
```

Comment:
I'm not sure if we want this to pass with an empty string or do we want it fail with a nullPointerException so we know that the transformation was not successful? This will not stop the flow of the request.

Reply:
I ended up using null